docs/api/ShallowWrapper/invoke.md

```diff
@@ -0,0 +1,39 @@
+# `.invoke(event[, ...args]) => Any`
+
+Invokes an event handler (a prop that matches the event name).
```

Comment:
i kind of wonder if maybe this should just be a prop name rather than an event name, ie, `onClick`, not `click`? Then it can be used to invoke any function-valued prop.

Reply:
Yes I think I agree

Reply:
+1 for `onClick` rather than `click`.